packages/core/src/index.ts

```diff
@@ -15,6 +15,7 @@ export * from "./callbacks/CallbackManager";
 export * from "./constants";
 export * from "./embeddings";
 export * from "./engines/chat";
+export * from "./extractors";
```

Comment:
Could we explicit export class from file instead of *.

I'm working on bundle splitting feature and this cause lots of confusion

Reply:
Maybe @yisding can jump in here as well, there's a PR that change. the another ones?

Reply:
@himself65 if you want to do it, let's do it in a different PR.

The reason I started with `export *` was for convenience and to prevent us from accidentally forgetting to export something. (even so sometimes we forget to export from a particular file and get a bug report).

If there are bundling reasons why we want to explicitly re-export every single export from every single file, let's find a way to write a test case or lint rule because otherwise I guarantee you we will to export a bunch of things.